stage1/init/common/pod.go

```diff
@@ -500,6 +451,81 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 	return nil
 }
 
+// parseUserGroup parses the User and Group fields of an App and returns its
+// UID and GID.
+// The User and Group fields accept several formats:
+//   1. the hardcoded string "root"
+//   2. a path
+//   3. a number
+//   4. a name in reference to /etc/{group,passwod} in the image
+// See https://github.com/appc/spec/blob/master/spec/aci.md#image-manifest-schema
+func parseUserGroup(p *stage1commontypes.Pod, ra *schema.RuntimeApp, privateUsers string) (int, int, error) {
+	app := ra.App
+	appName := ra.Name
+
+	var uid_, gid_ int
```

Comment:
Using `u` and `g` here is a bit too short for being variables that are used all over the place in this function. I think `uid_`  and `gid_` is better.